utils/build/docker/python/flask/app.py

```diff
@@ -102,8 +102,8 @@
     level=logging.INFO,
     format=(
         "%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] "
-        "[dd.service=%(dd.service)s dd.env=%(dd.env)s dd.version=%(dd.version)s dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] "
-        "- %(message)s"
+        "[dd.service=%(dd.service)s dd.env=%(dd.env)s dd.version=%(dd.version)s dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s]"
+        " %(message)s"
```

Comment:
is it a leftover ?

Reply:
Nope. This aligns the python log injection format with what ruby generates by default. Making this change helps simplify the tests